commit daec6ebf58331776279f389738af4788b865033d
Author: Michael Schroeder <mls@suse.de>
Date:   Mon Apr 12 12:10:20 2010 +0200

    readLineFromOFI may modify the fileStack, thus we have to re-set
    ofi after calling it

diff --git a/build/parseSpec.c b/build/parseSpec.c
index 06b5a3d..f45022b 100644
--- a/build/parseSpec.c
+++ b/build/parseSpec.c
@@ -274,6 +274,7 @@ int readLine(rpmSpec spec, int strip)
     retry:
 	if ((rc = readLineFromOFI(spec, ofi)) != 0)
 	    return rc;
+	ofi = spec->fileStack;
 
 	/* Copy next file line into the spec line buffer */
 	rc = copyNextLineFromOFI(spec, ofi);
